src/components/NavBar/NavBar.tsx

```diff
@@ -4,31 +4,47 @@ import AccountCircleIcon from "@mui/icons-material/AccountCircle";
 import HomeOutlinedIcon from "@mui/icons-material/HomeOutlined";
 import MailOutlinedIcon from "@mui/icons-material/MailOutlined";
 import AccountCircleOutlinedIcon from "@mui/icons-material/AccountCircleOutlined";
-import { Avatar, Box, Button, List, Toolbar } from "@mui/material";
+import { Box, Button, IconButton, List, Toolbar } from "@mui/material";
 import AccountMenu from "./AccountMenu";
 import NavItem from "./NavItem";
 import PostButtonModal from "./PostButtonModal";
 import { useState } from "react";
 import ComposePost from "../Posts/ComposePost";
 import { useAppSelector } from "../../state/hooks";
 import { useLocation } from "react-router-dom";
+import { Link as Routerlink } from "react-router-dom";
 
 const styles = {
   icon: {
     color: "black.main",
     opacity: 0.9,
   },
+  iconButton: {
+    alignSelf: "flex-start",
+    "&.Mui-selected": {
```

Comment:
Do we use this for the iconButton?

Reply:
Yup you're right, I just copy pasted code without checking lol